### PR TITLE
Added code to handle response codes from ays

### PR DESCRIPTION
**Pull request by abdulrahmantkhalifa**
relates to issue #362

### CODE REVIEW

File: api/node/node_api_CreateContainer.go [hunk 1]
```diff
@@ -35,9 +35,11 @@ func (api NodeAPI) CreateContainer(w http.ResponseWriter, r *http.Request) {
 	// validate container name
 	exists, err := tools.ServiceExists("container", reqBody.Name, api.AysRepo)
 	if err != nil {
+		fmt.Println("wtv the fuck 38")
```
Comment (serboctor) [inline api/node/node_api_CreateContainer.go]: ??

File: api/node/node_api_CreateContainer.go [hunk 2]
```diff
@@ -35,9 +35,11 @@ func (api NodeAPI) CreateContainer(w http.ResponseWriter, r *http.Request) {
...
 		tools.WriteError(w, http.StatusInternalServerError, err)
 		return
 	} else if exists {
+		fmt.Println("wtv the fuck 42 ")
```
Comment (serboctor) [inline api/node/node_api_CreateContainer.go]: ??

Comment (abdulrahmantkhalifa) [inline api/node/node_api_CreateContainer.go]: sorry about that

**Comment by grimpy** [comment]
Lgtm

**Pull request merged**